### PR TITLE
Add a few tests for the customviewstate3dcreator

### PR DESCRIPTION
**Pull request by nick4598**
Fixes https://github.com/iTwin/itwinjs-backlog/issues/485

### CODE REVIEW

File: common/changes/@itwin/core-backend/nick-add-customviewstate3dcreator-tests_2022-11-17-14-10.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/core-backend",
+      "comment": "add customviewstate3dcreator test",
```
Comment (aruniverse) [inline common/changes/@itwin/core-backend/nick-add-customviewstate3dcreator-tests_2022-11-17-14-10.json:5]:
```suggestion
      "comment": "",
```

File: full-stack-tests/core/src/frontend/standalone/ViewCreator3d.test.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    const div = document.createElement("div");
+    div.style.width = div.style.height = "20px";
+    document.body.appendChild(div);
+    const viewcreator3d = new ViewCreator3d(imodel);
+    const viewState = await viewcreator3d.createDefaultView();
+    expect(viewState).to.exist.and.be.not.empty;
+
+    const testVp: ScreenViewport = ScreenViewport.create(div, viewState);
+    await testVp.waitForSceneCompletion();
+    if (ProcessDetector.isElectronAppFrontend) {
```
Comment (aruniverse) [inline full-stack-tests/core/src/frontend/standalone/ViewCreator3d.test.ts:33]: just curious, why is this different between electron and browser?

**Review by aruniverse — APPROVED** [review]

File: full-stack-tests/core/src/frontend/standalone/ViewCreator3d.test.ts (hunk 2 above)
Comment (nick4598) [reply full-stack-tests/core/src/frontend/standalone/ViewCreator3d.test.ts:33]: I don't have a good technical explanation for that, it seemed consistent atleast. Slight differences between rendering tiles in electron vs browser? Maybe Paul or mark would be able to explain.

Comment (aruniverse) [reply full-stack-tests/core/src/frontend/standalone/ViewCreator3d.test.ts:33]: @pmconne @markschlosseratbentley ?

I wont hold the pr for this; but just curious to know

Comment (nick4598) [reply full-stack-tests/core/src/frontend/standalone/ViewCreator3d.test.ts:33]: I tried 400px and my test still passed (meaning electron was 7 tiles still). It also passed at 800px. I am running build:ci (almost fell into the trap of just running build).

Comment (pmconne) [reply full-stack-tests/core/src/frontend/standalone/ViewCreator3d.test.ts:33]: I guess I'll pull your branch and see what's up with the discrepancy.

Comment (pmconne) [reply full-stack-tests/core/src/frontend/standalone/ViewCreator3d.test.ts:33]: Fixed.

**Review by pmconne — APPROVED** [review]

**Pull request merged**